text/0000-mixed-shadow-mode.md

```diff
@@ -0,0 +1,206 @@
+implemented as patches on global prototypes so the decision of invoking a polyfilled API vs a native
+API will depend on the preference of individual components during runtime. This test environment
+will ensure that components designed to run in a native Shadow DOM environment will work correctly
+when even when synthetic Shadow DOM polyfills are present.
+
+Components that prefer native mode will need to also support synthetic mode in browser environments
+where Shadow DOM is natively unavailable, so they will also need to be tested in both modes. LWC
+test utilities will need to be updated to facilitate this.
+
+The existing WPT (Web Platform Tests) test suite for Shadow DOM APIs will also be run to identify
+any coverage gaps in LWC integration tests.
+
+## Observable differences
+
+### Light DOM and assigned elements
+
+In native Shadow DOM, elements are rendered in a component's light DOM and assigned to slots in a
+child component's shadow DOM. This means that elements will exist in the DOM regardless of whether
+they are assigned to a slot. This is not the case for LWC--elements that are passed down to child
+components are never rendered unless they are assigned to a slot.
+
+In the following example, `span` will not exist in the DOM in synthetic shadow, but it will exist in
+native shadow.
+
+```html
+<!-- x-parent -->
+<template>
+    <x-child>
+        <span>foo</span>
+    </x-child>
+</template>
+
+<!-- x-child -->
+<template>
+    <p>child</p>
+</template>
+```
+
+This observable difference will not be remediated as such a change would be non-trivial and
+backwards-incompatible with the current synthetic shadow implementation.
+
+### Lifecycle timing
+
+Due to the way that slotting is implemented in `@lwc/synthetic-shadow`, there is an observable
+difference in the timing of lifecycle hooks for slotted elements.
+
+As mentioned previously, in synthetic mode, slotted elements that are never assigned to a slot are
+not rendered. This means that their lifecycle hooks are never invoked.
+
+Another difference is that for synthetic mode, lifecycle hooks are invoked in the order of
+appearance after they are assigned, whereas in native mode, they are invoked in the order of
+appearance in the template.
+
+### this.shadowRoot vs this.template
+
+`this.shadowRoot` and `this.template` will both reference the component's shadow root in native
+Shadow DOM while `this.template` currently references the component's shadow root in synthetic
+Shadow DOM.  When a component specifies a preference for native Shadow DOM, `this.shadowRoot` will
+be made available as an alias to `this.template`.
+
+There are no plans to enable `this.shadowRoot` for existing components that only support synthetic
+Shadow DOM as the current LWC-ism is to use `this.template`.
+
+### this.shadowRoot instanceof ShadowRoot
+
+Components that favor native Shadow DOM will observe that `this.shadowRoot instanceof ShadowRoot`

```

Comment:
If this will be remediated, which it should, why are we arguing that this statement is false?